core/src/util/mod.rs

```diff
@@ -24,6 +28,63 @@ lazy_static! {
     static ref NON_ESCAPED_PERCENT: FancyRegex = FancyRegex::new(r"(?<!\\\\)%").unwrap();
 }
 
+pub(crate) fn handle_libmongosqltranslate_command(command: Document) -> Result<Document> {
```

Comment:
[nit] I think that the function name is not bad, except that because it is the ODBC driver I would rather stay clear of using the word `handle` if possible 😅
```suggestion
pub(crate) fn libmongosqltranslate_runCommand(command: Document) -> Result<Document> {
```